Pipfile

```diff
@@ -34,6 +34,7 @@ twine = "==4.0.1"
 types-six = "==1.16.21.4"
 dagit = "==1.2.6"
 dagster = "==1.2.6"
+dill = "==0.3.7"
```

Comment:
why's this?

Reply:
Because pylint doesn't work without it, if you know a better solution please let me know. The only other thing I could do was changing the lock file manually, but that didn't seem like the right approach.

Reply:
🤔 curious. The dill dependency was already present in the `pipfile.lock`. Pylint worked with 3.11 on my machine.

Reply:
It also worked on my machine, but not in the github action

Reply:
Interesting. Maybe you can explain that in a comment with the depedency

Reply:
This is also fixed with the change below